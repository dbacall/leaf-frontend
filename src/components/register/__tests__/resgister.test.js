import React from 'react';
import { shallow } from 'enzyme';
import Register from '../registerComponent';

describe('Register component tests:', () => {
  const props = {
    errors: {},
  };

  it('should render correctly', () => {
    const wrap = shallow(<Register {...props} />);
    expect(wrap).toMatchSnapshot();
  });
});
