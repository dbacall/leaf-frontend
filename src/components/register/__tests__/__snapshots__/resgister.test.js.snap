// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register component tests: should render correctly 1`] = `
<section
  className="register"
>
  <div
    className="formContainer"
  >
    <div>
      <h4>
        Register
      </h4>
      <p>
        Already have an account? 
        <Link
          to="/login"
        >
          Log in
        </Link>
      </p>
    </div>
    <form
      noValidate={true}
      onSubmit={[Function]}
    >
      <input
        className=""
        id="firstName"
        onChange={[Function]}
        placeholder="First Name"
        type="text"
        value=""
      />
      <p
        className="warning"
      />
      <input
        className=""
        id="surname"
        onChange={[Function]}
        placeholder="Surname"
        type="text"
        value=""
      />
      <p
        className="warning"
      />
      <input
        className=""
        id="email"
        onChange={[Function]}
        placeholder="Email"
        type="email"
        value=""
      />
      <p
        className="warning"
      />
      <input
        className=""
        id="password"
        onChange={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
      <p
        className="warning"
      />
      <input
        className=""
        id="password2"
        onChange={[Function]}
        placeholder="Confirm Password"
        type="password"
        value=""
      />
      <p
        className="warning"
      />
      <button
        className="submitBtn"
        type="submit"
      >
        Sign up
      </button>
    </form>
  </div>
</section>
`;
